application/utils/msgparse.py

```diff
@@ -0,0 +1,53 @@
+from cmd import Cmd
+
+class MsgParse(Cmd):
+
+    def precmd(self, line):
+        return line.lower()
```

Comment:
Might also want to add whitespace removal. line.strip() ?

Reply:
Good call. Done in 7b584ae.